testcases/OpTestHtxBootme.py

```diff
@@ -469,53 +452,4 @@ def get_mpath_from_dm(self, dm_id):
             raise MPException(f"Multipathd Command Failed : {ex} ")
         for mpath in mpaths:
             if dm_id in mpath:
-                return mpath.split()[1]
-
-    def get_all_disk_paths(self):
-        """
-        Returns all available disk names and alias on this  system
-
-        This will get all the sysfs disks name entries by its device
-        node name, by-uuid, by-id and by-path, irrespective of any
-        platform and device type
-
-        :returns: a list of all disk path names
-        :rtype: list of str
-        """
-        disk_list = abs_path = []
-        for path in [
-            "/dev",
-            "/dev/mapper",
-            "/dev/disk/by-id",
-            "/dev/disk/by-path",
-            "/dev/disk/by-uuid",
-            "/dev/disk/by-partuuid",
-            "/dev/disk/by-partlabel",
-        ]:
-            if self.execute_remote_command('test -e {}'.format(path)):
-                directory = self.execute_remote_command('ls -l {}'.format(path))
-                for device in directory:
-                    abs_path.append(path +  '/' + device)
-                disk_list.extend(abs_path)
-        return disk_list
-
-    def get_absolute_disk_path(self, device):
-        """
-        Returns absolute device path of given disk
-
-        This will get actual disks path of given device, it can take
-        node name, by-uuid, by-id and by-path, irrespective of any
-        platform and device type
-
-        :param device: disk name or disk alias names sda or scsi-xxx
-        :type device: str
-
-        :returns: the device absolute path name
-        :rtype: bool
-        """
-        if not self.execute_remote_command('test -e {}'.format(device)):
-            for dev_path in self.get_all_disk_paths():
-                dev_base = self.execute_remote_command('basename $(realpath {})'.format(dev_path))
-                if device == dev_base:
-                    return dev_path
-        return device
```

Comment:
@abdhaleegit I am removing these functions as i am directly running commands on host and getting the device paths, Tested this by providing 2 different paths as '/dev/mapper/mpatha' and '/dev/sda' it worked fine in both the cases.